scripts/fs.py

```diff
@@ -76,51 +94,57 @@ def main(self):
                 logging.info('Mask the t1')
                 t1 = tmpdir / 't1masked.nii.gz'
                 ImageMath('3', t1, 'm', self.t1, self.t1mask)
-                skullstrip = '-noskullstrip'
 
             else:
-                skullstrip = '-skullstrip'
                 t1 = tmpdir / 't1.nii.gz'
                 self.t1.copy(t1)
             
+            subjid = self.t1.stem
+            common_params=['-s', subjid]
+
+
+            autorecon3_params=[]
             if self.t2:
                 if self.t2mask:
                     logging.info('Mask the t2')
                     t2 = tmpdir / 't2masked.nii.gz'
                     ImageMath('3', t2, 'm', self.t2, self.t2mask)
-                    skullstrip = '-noskullstrip'
-
+            
                 else:
-                    skullstrip = '-skullstrip'
                     t2 = tmpdir / 't2.nii.gz'
-                    self.t2.copy(t2)
+                    self.t2.copy(t2)  
+                
+                autorecon3_params= ['-T2', t2, '-T2pial']
+            
```

Comment:
-T2 is used for -T2pial only. @tashrifbillah checked it with freesurfer developers.